packages/jetpack-redirects/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "@automattic/jetpack-redirects",
+	"version": "1.0.0",
+	"description": "Jetpack Redirects helper",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": false,
+	"keywords": [
+		"wordpress",
+		"jetpack"
+	],
+	"author": "Automattic Inc.",
+	"contributors": [
+		"leogermani"
+	],
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/jetpack-redirects"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
```

Comment:
The `files` field specifies which files and directories are distributed when publishing to NPM. Do we need to include the `src/` dir? Shouldn't `dist/` be enough?

Instead, the `files` list should include `README.md`.

I see the blueprint in `docs/monorepo.md` recommends this 🙁 @sirreal what do you think? Are there some TypeScript reasons for distributing `src/` maybe?

Reply:
No TypeScript reasons for distributing `src` that I can think of.

I spotted this in another review: https://github.com/Automattic/wp-calypso/pull/42409#discussion_r428606593

The docs do need an update. Do you think it's worth even including `files`? In my experience it leads to bugs that you don't discover until after publish and the benefit is pretty limited.

Reply:
> Do you think it's worth even including `files`?

I've seen some popular packages shipping a megabyte of test fixtures, although the real code is just a few lines. Other than that, yeah, the benefits are limited. I personally still prefer minimalistic, zero-bloat software though 🙂

Reply:
done in https://github.com/Automattic/wp-calypso/pull/41497/commits/88b5240349ffc28170276a3f9165ea482bf9ad79